css/navigation.css

```diff
@@ -7,6 +7,9 @@
  * @author Bernhard Posselt <dev@bernhard-posselt.com>
  * @copyright Bernhard Posselt 2014
  */
+#app-navigation {
+    overflow-y: hidden !important;
```

Comment:
Wouldn't this mean that a long list of folders would hide behind the settings?

Reply:
Both sections (folders/feeds and settings) scroll independently in that scenario:

![Recording 2022-10-23 at 22 09 23](https://user-images.githubusercontent.com/5313660/197418397-bcc456cd-cb9c-45ea-a6c4-4d0b3c25fae7.gif)